docs/UserGuide.md

```diff
@@ -629,22 +680,22 @@ Format: `list`
 
 ### Finding patients: `find`
 
-You can find specific patients using the `find` command.
+Finds patients whose parameters contain the given keywords.
 
-Format: **`find`**`[KEYWORD]… [xyz/XYZ_KEYWORD]…`
+Format: `find [KEYWORD]… [xyz/XYZ_KEYWORD]…`
```

Comment:
Similar to above, a change I suggested and you fixed appears to have been reverted